src/backend/drm/DRM.cpp

```diff
@@ -269,7 +306,16 @@ std::vector<SP<CDRMBackend>> Aquamarine::CDRMBackend::attempt(SP<CBackend> backe
 }
 
 Aquamarine::CDRMBackend::~CDRMBackend() {
-    ;
+    for (auto conn : connectors) {
+        if (conn && conn->output)
+            conn->output->events.destroy.emit();
+        conn.impl_->destroy();
```

Comment:
wait, wtf is this? These pointers are managed, don't do this.

just conn.reset()